jenkins/sparklingWaterPipeline.groovy

```diff
@@ -115,17 +101,21 @@ def withSharedSetup(sparkMajorVersion, config,  shouldCheckout, code) {
 
                 config.put("sparkVersion", getSparkVersion(config))
 
+                if (config.buildAgainstH2OBranch.toBoolean()) {
+                    config.put("driverJarPath", "${env.WORKSPACE}/h2o-3/h2o-hadoop-2/h2o-${config.driverHadoopVersion}-assembly/build/libs/h2odriver.jar")
+                } else {
+                    def majorVersionLine = readFile("gradle.properties").split("\n").find() { line -> line.startsWith('h2oMajorVersion') }
```

Comment:
Isn't there any way how to parse `gradle.properties` to  a map and then extract a given values from the map?

Reply:
Yes, that would be possible

Reply:
Will do it right after this PR, have JIRA for it https://0xdata.atlassian.net/browse/SW-1909

It will be nice clean-up as well, but don't want to make this change even bigger. Is it ok with you ? :)

Reply:
ok, no problem